### ISSUE TITLE
Unable to find type [System.Windows.Media.Stretch]

### ISSUE THREAD

**Issue by Agazoth** [jdhitsolutions/PSCalendar#27]
# Problem Report

Show-GuiCalendar does not work (on my machine)

## What is the syntax or expression you are using?
Show-GuiCalendar 12/2018 2/2019 -highlight 12/24/18,12/25/18,12/31/18,1/1/19,1/18/19,2/14/19,2/22/19

## What Happens?
```powershell
InvalidOperation: C:\Users\axel\Documents\PowerShell\Modules\PSCalendar\2.3.2\functions\public.ps1:160
Line |
 160 |          [System.Windows.Media.Stretch]$Stretch = "UniformToFill",
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Unable to find type [System.Windows.Media.Stretch].



Unable to find type [System.Windows.Media.Stretch].
At C:\Users\axel\Documents\WindowsPowerShell\Modules\PSCalendar\2.3.2\functions\public.ps1:160 char:9
+         [System.Windows.Media.Stretch]$Stretch = "UniformToFill",
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Windows.Media.Stretch:TypeName) [], RuntimeException
    + FullyQualifiedErrorId : TypeNotFound
```
## What do you think should happen?

A GUI calendar should open

## Environment Information

```powershell
Name                           Value
----                           -----
PSVersion                      5.1.22000.282
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22000.282
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

Name                           Value
----                           -----
PSVersion                      7.1.5
PSEdition                      Core
GitCommitId                    7.1.5
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0


PS> $PSCulture
da-DK

PS> $PSUICulture
en-US

PS> Get-Module PSCalendar

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     2.3.2                 PSCalendar                          {Get-Calendar, Get-MonthName, Get-NCalendar, Get-P…

```

**Comment by jdhitsolutions** [comment]
It is very possible that the necessary .NET classes are not part of .NET core in PowerShell 7. I'm assuming if you run the command in Windows PowerShell it works just fine.  I may have to refactor the module not to export `Show-GUICalendar` for PowerShell 7.

**Comment by jdhitsolutions** [comment]
Looks like a limitation in PowerShell 7. The .NET class works fine in Windows PowerShell.
![image](https://user-images.githubusercontent.com/12089920/140062352-667a0e16-77d7-431a-bed7-940a3be4697b.png)

But is missing in PowerShell 7
![image](https://user-images.githubusercontent.com/12089920/140062388-3361d9e3-b042-430e-8117-a3ebfa8b687e.png)

**Comment by jdhitsolutions** [comment]
Because this is a .NET limitation, obviously I can't fix it. I've updated the module and v2.4.0 no longer exports this command in PowerShell 7. That's the best I can do.

**Comment by Agazoth** [comment]
It is actually the same in Windows Powershell. I included the error from both.

**Comment by jdhitsolutions** [comment]
I missed that. That shouldn't be the case. I have never had an issue in Windows PowerShell (testing on multiple systems), but I got the same problem on PowerShell 7.  Given that .NET Core is a subset of the  .NET Framework in Windows PowerShell, this doesn't surprise me.

I can't think why this would fail in Windows PowerShell. Are you running something like a Home or Student edition? Not that it should make a difference.  I can't imagine the Danish culture is causing a problem. 

Update the module to 2.4.0 and try again in Windows PowerShell. I changed a parameter to use a string instead of the .NET class. Let's see if you get the same error. The command definitely won't run in PowerShell 7.
